### PR TITLE
Task Order 24 cleaned changes

### PR DESCRIPTION
**Pull request by rlevy12**
This PR represents the cleaned-up and combined changes to ItemRenderer with respect to Task Order 24. This is done to eliminate the noise caused by auto formatting in earlier versions of these changed files, which complicated comparing and merging.

### CODE REVIEW

File: TDS.ItemRenderer/src/main/java/tds/blackbox/ResourcesSingleton.java [hunk 1]
```diff
@@ -154,11 +196,14 @@ public static String resolveUrl (String virtualPath, boolean includeCacheID, boo
...
   public static List<String> getFilePaths (String resourceFileVirtualPath, String resourceID, boolean addCRC) throws URISyntaxException, ResourcesException {
+		String resourceFileDir = Path.getDirectoryName(resourceFileVirtualPath);
+		String rootVirtualPath = Server.mapPath("~/");
     List<String> returnList = new ArrayList<String> ();
-    String resourceFileName = Path.getFileName (resourceFileVirtualPath);
+		// String resourceFileName = Path.getFileName (resourceFileVirtualPath);
 
     // e.x. "~/shared/"
-    String resourceVirtualPath = StringUtils.replace (resourceFileVirtualPath, resourceFileName, "");
+		// String resourceVirtualPath = StringUtils.replace
```
Comment (hbgwhite) [inline TDS.ItemRenderer/src/main/java/tds/blackbox/ResourcesSingleton.java]: Please remove commented code

File: shared-blackbox/src/main/webapp/Scripts/HTMLEditor2/frame.css [hunk 2]
```diff
@@ -1,152 +1,148 @@
...
-	position: fixed;
-	top: 0px;
-	left: 0px;
-	bottom: 0px;
-	right: 0px;
-	-moz-box-sizing: content-box;
-	box-sizing: content-box;
-}*/
\ No newline at end of file
+/* @group Reset Code */
```
Comment (hbgwhite) [inline shared-blackbox/src/main/webapp/Scripts/HTMLEditor2/frame.css]: We cannot tell what changed here - it appears that the whole file was formatted. Please resubmit.

File: shared-blackbox/src/main/webapp/Scripts/Scaffolding/asi.css [hunk 3]
```diff
@@ -1,271 +1,261 @@
...
-
-/* ASI table stem CSS fixes*/
-.asi-stem table{
-  max-width: 100%;
-}
-
-.asi-stem table p{
-	padding: 0.2em;
-}
+.asi-container {
```
Comment (hbgwhite) [inline shared-blackbox/src/main/webapp/Scripts/Scaffolding/asi.css]: We cannot tell what changed here - it appears that the whole file was formatted. Please resubmit.

File: shared-blackbox/src/main/webapp/Scripts/Simulator2/Renderer/CSS/simulator.css [hunk 4]
```diff
@@ -1,1637 +1,1641 @@
...
-.layout_wai .simulator .inputPanel .inputpanelrow {
-	margin-left: 5px;
-}
-.layout_wai .simulator .dataOutputPanel table {
-	margin-top: 10px;
-}
-.layout_wai .simulator .table_controls .actionButton {
-    font-size: 1em;
-    margin-bottom: 5px;
+.simulator {
```
Comment (hbgwhite) [inline shared-blackbox/src/main/webapp/Scripts/Simulator2/Renderer/CSS/simulator.css]: We cannot tell what changed here - it appears that the whole file was formatted. Please resubmit.

Comment (vkesharia) [inline shared-blackbox/src/main/webapp/Scripts/Simulator2/Renderer/CSS/simulator.css]: Please refer to commit done today
https://github.com/SmarterApp/TDS_ItemRenderer/commit/4b46e6cc34734c4c54d35479a919e7c050f9fece

File: shared-blackbox/src/main/webapp/Scripts/Scaffolding/asi.css (hunk 3 above)
Comment (vkesharia) [inline shared-blackbox/src/main/webapp/Scripts/Scaffolding/asi.css]: Please refer to commit done today
https://github.com/SmarterApp/TDS_ItemRenderer/commit/4b46e6cc34734c4c54d35479a919e7c050f9fece

File: shared-blackbox/src/main/webapp/Scripts/HTMLEditor2/frame.css (hunk 2 above)
Comment (vkesharia) [inline shared-blackbox/src/main/webapp/Scripts/HTMLEditor2/frame.css]: Please refer to commit done today
https://github.com/SmarterApp/TDS_ItemRenderer/commit/4b46e6cc34734c4c54d35479a919e7c050f9fece

File: TDS.ItemRenderer/src/main/java/tds/blackbox/ResourcesSingleton.java (hunk 1 above)
Comment (vkesharia) [inline TDS.ItemRenderer/src/main/java/tds/blackbox/ResourcesSingleton.java]: https://github.com/SmarterApp/TDS_ItemRenderer/commit/727da933ec281912d3eba681973caf18a31f786a

**Pull request merged**